### PR TITLE
Create Blog “streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics”

### PR DESCRIPTION
**Pull request by joann-starke**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: "Streamline and optimize ML workflows with HPE Ezmeral Unified Analytics "
+date: 2023-09-27T13:00:00.000Z
+author: "Abhishek Kumar Agarwal "
+authorimage: /img/abbie-192x192.png
+thumbnailimage: /img/developers-512x400.png
+disable: false
+tags:
+  - "opensource hpe-ezmeral data-ml-engineer data-scientist "
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:9]: Hi Joann, Abhishek,

For Tag, please make sure to use separate tags (in the CMS editor, at the bottom of the editor, click "Add Tags" to include individual tag):
- opensource
- hpe-ezmeral
- data-ml-engineer
- data-scientist

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+---
+<!--StartFragment-->
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:11]: In the CMS Editor, in Markdown mode, please delete this XML code \<!--StartFragment--\>

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+This is part two of a blog series that showcases the capabilities of [HPE Ezmeral Unified Analytics](https://www.hpe.com/us/en/hpe-ezmeral-unified-analytics.html) through the real-world example of stock market predicting. In our [previous blog](https://developer.hpe.com/blog/seamless-data-engineering-for-financial-services/), we covered the data engineering aspects of Apache Spark and Superset to streamline pipelines and visualize data. In this blog, we will continue with the same use case, stock market forecasting, to showcase how HPE Ezmeral simplifies building, deploying, and productizing machine learning models and pipelines.
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:13]: The URL https://www.hpe.com/us/en/hpe-ezmeral-unified-analytics.html give an error message. Could you please confirm the URL?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+---
+<!--StartFragment-->
+
+This is part two of a blog series that showcases the capabilities of [HPE Ezmeral Unified Analytics](https://www.hpe.com/us/en/hpe-ezmeral-unified-analytics.html) through the real-world example of stock market predicting. In our [previous blog](https://developer.hpe.com/blog/seamless-data-engineering-for-financial-services/), we covered the data engineering aspects of Apache Spark and Superset to streamline pipelines and visualize data. In this blog, we will continue with the same use case, stock market forecasting, to showcase how HPE Ezmeral simplifies building, deploying, and productizing machine learning models and pipelines. 
+
+**Model Building and Training**
+
+Business problem: Forecast stock prices of different companies listed in the National Stock Exchange (NSE) of India.
+
+Step 1: Data Gathering\
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:19]: Please remove the backslash after Step 1: Data Gathering; Step 2: Data Preprocessing, Step3: Modeling, 
Just insert a carriage return between the Step and the description sentence.

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+Step 3: Modeling\
+First, the data is divided between training and validation and then used to train a long short term memory (LSTM) model. After training, the model is evaluated on the error metric.
+
+LSTM is a variety of [recurrent neural networks (RNNs)](https://www.techtarget.com/searchenterpriseai/definition/recurrent-neural-networks#:~:text=A%20recurrent%20neural%20network%20is,predict%20the%20next%20likely%20scenario.) capable of learning long-term dependencies, especially in sequence prediction problems.  
+
+**Model Tracking and Monitoring using MLflow**
+
+MLflow is an open-source platform designed to manage the end-to-end machine learning lifecycle. It provides tools for tracking experiments, packaging code, sharing models, and managing deployment. MLflow was developed by Databricks, a company that specializes in big data and AI. They have gained popularity within the machine learning community for their ability to streamline and organize various stages of machine learning workflows.
+
+MLflow consists of several key components:\
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:36]: Please remove the backslash after MLFlow consists of several key components:

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+\
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:37]: Delete the backslash, and just add a carriage return.

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+Next, script the KServe yaml file to begin the inference service. 
+
+![](/img/fig-1b-inference-service.png "Figure 9. The inference service is defined. ")
+
+Next, a new model server must be created in Kube Flow and the yaml file uploaded. The inference service is created and once it is running, we can send our test data through REST APIs and get the response.
+
+**Model Retraining**
+
+The models deployed into production are constantly monitored to identify inconsistencies and degradation in performance. Because models can skew to the old data used for training resulting in under performance, models should be retrained whenever performance drops below a defined threshold. To retrain a model, a pipeline is developed that could trigger all the steps associated with model retraining to occur in the appropriate sequence using Kube Flow pipelines.
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:109]: In this section, please check occurrences of "Kube Flow": Is it Kube Flow or Kubeflow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+Once each of the tasks are defined, the Kube Flow pipelines are defined to call each of the steps using DSL (Domain Specific Language).
+
+![](/img/fig3-define-the-pipeline.png "Figure 12. Pipelines are defined along with tasks in the appropriate sequence. ")
+
+When the Kube Flow pipeline is set up, the pipeline can run using the Kube Flow client where the required arguments and parameters are passed for the specific run.
+
+![](/img/kubeflow-setup-run.png "Figure 13. Kube Flow client is created and executed with appropriate arguments. ")
+
+In the Kube Flow UI, “stock_pipeline_run” would be triggered to provide a green-red status of each step as it executes.
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:129]: Use escape character for the underscore in the "stock_pipeline_run":
stock\\_pipeline\\_run

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+
+In the Kube Flow UI, “stock_pipeline_run” would be triggered to provide a green-red status of each step as it executes. 
+
+![](/img/fig-5-simplified-pipeline.png "Figure 14. Status of each pipeline step and it's status. ")
+
+**Conclusion**
+
+From the above example, HPE Ezmeral Unified Analytics has significantly enhanced the end-to-end pipeline of financial service use cases by providing a comprehensive platform tailored to the specific needs of ML practitioners and data scientists. Here is a conclusion highlighting the benefits of HPE Ezmeral Unified Analytics:
+
+Seamless Integration: HPE Ezmeral Unified Analytics offers seamless integration of various data engineering and ML components like Spark, EzPresto, Superset, MLflow, Kube Flow. This integration streamlines the entire ML workflow, reducing friction between dissimilar stages and enables more efficient development.
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:137]: In the Conclusion section, is it Kube Flow or Kubeflow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+
+Data Exploration and Visualization: By seamlessly integrating Apache Superset into the Ezmeral ecosystem, organizations can harness the power of data-driven insights. Teams can easily access and visualize data, track experiment results, and make data-informed decisions to improve model quality and performance.
+
+Reproducibility: Kube Flow ensures reproducibility by allowing users to define and version their entire ML pipelines using HPE Ezmeral Unified Analytics. This means that every step, from data preparation to model deployment, can be tracked, versioned, and replicated easily, leading to better model accountability and auditability.
```
Comment (choukden) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:141]: For Reproducibility, do you mean Kube Flow or MLFlow?

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Joann, Abhishek, 

Thank you very much for your second post of the blog series about HPE Ezmeral UA. This is a great series.

Here are some "technical" feedback.
Dale will go through the editorial review in the coming day or so. So expect additional suggestions for edit in the coming days.

Here are some tips to assist you with the review of the suggestions and how to acknowledge them in GitHub:

- Navigate to **File Changed** area to visualize the comments from start to the end,or navigate to **Conversation area** to visualize the comments per reviewer.

- As you incorporate the suggestions for edit in your blog post, please click on button **Resolve Conversation** in GitHub for each suggestion you incorporated.

Note we recommend you to continue doing edits of the blog post entry in the CMS Workflow editor while the entry is in the "**In Review"** column. Moving it back to the "**Drafts**" column will close the Pull Request (PR) in GitHub and when moving the entry to the “In Review” column again a new PR in GitHub is created, and we will lose the suggestions for edits.

Thanks again, 
Best regards,
Denis

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+
+To begin, allow the experiment to access MLflow from the Jupyter notebook. Then as we have used TensorFlow LSTM as our ML model, we use mlflow.tensorflow.autolog() to log automatically the training parameters, hyperparameters and model metrics.
+
+![](/img/figure-1a-experiment.png "Figure 2. Creating an experiment with auto logging. ")
+
+Once the model is ready, the run details are available via MLflow APIs like search_run(). The best run can be auto-selected as per our requirements.
+
+![](/img/fig-1b-best-run-experiment.png "Figure 3. Selecting the best run of the experiment and loading the model.")
+
+On the Web UI for MLflow, we can access multiple runs of the ML experiment for different parameter combinations. Details of the run can be accessed by clicking on the corresponding run name.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:60]: Change "we" to "you" in the first sentence. "We" don't use the "royal we" :-) in our blog posts.

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+
+![](/img/fig-2-multiple-runs.png "Figure 4. Results from multiple runs made on the experiment with different parameters in each run. ")
+
+Model performance can be obtained by comparing various runs. In the image below, all six runs have been selected and we see that the model with epochs=20, performs best with the least loss and mean absolute error (MAE).
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:64]: Change "we see" to "you can see"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 13]
```diff
@@ -0,0 +1,153 @@
...
+
+![](/img/fig-3-multiple-runs-compared.png "Figure 5. Comparison of all runs based on a relevant metric. ")
+
+Now that the best model has been identified, it can be registered programmatically or through the MLflow Web UI. Once registered, the model can be deployed to the appropriate staging-production region. 
+
+![](/img/fig-4-model-registry-different-models.png "Figure 6. Model registry with different models. ")
+
+**Code Repository**
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:72]: Change to "Code respository"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 14]
```diff
@@ -0,0 +1,153 @@
...
+
+A code repository is a storage location for code and other software development assets, such as documentation, tests, and scripts. They are often used to manage and organize a software project's codebase and collaborate with other project developers.
+
+The source code for the model is stored in a private repository on GitHub then seamlessly pulled into the IDE using git commands. The updated model can be pushed to different branches of the source repo. 
+
+![](/img/code-repository.png "Figure 7.  Source code for models is stored on Github in a private repository. ")
+
+**Model Deployment**
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:80]: Change to "Model deployment"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+
+The next step is to deploy the best model chosen from the MLflow experiment. Here we are using KServe which is a standard Model Inference Platform on Kubernetes, built for highly scalable use cases.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:82]: Change the second sentence to read:
Here, Kserve, which is a standard model inference platform on Kubernetes that is build for highly scalable use cases, is being used. (I know... many prefer the active tense, but sometimes it's hard to get around the Royal We.

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 16]
```diff
@@ -0,0 +1,153 @@
...
+
+Kube Flow Serving, also known as KServe, is a component of the Kube Flow ecosystem and is designed to serve machine learning models in Kubernetes environments. Kube Flow itself is an open-source platform for deploying, monitoring, and managing machine learning models on Kubernetes.
+
+KServe provides several features and benefits:
+
+1. Scalability: It is designed to handle serving machine learning models at scale. You can easily scale up or down based on the traffic and load requirements.
+2. Multi-Framework Support: KServe supports serving models trained in various machine learning frameworks, such as Tensor Flow, PyTorch, scikit-learn, and others.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:89]: Use "Multi-framework support:"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 17]
```diff
@@ -0,0 +1,153 @@
...
+3. Advanced Deployment Strategies: KServe supports various deployment strategies, for example  Canary deployments or Blue-Green deployments,  allowing you to roll out new model versions gradually and monitor their performance.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:90]: Use "Advanced deployment strategies:"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 18]
```diff
@@ -0,0 +1,153 @@
...
+4. Monitoring and Metrics: It provides metrics and monitoring capabilities, allowing you to keep track of model performance, errors, and other relevant statistics.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:91]: Use "Monitoring and metrics:"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 19]
```diff
@@ -0,0 +1,153 @@
...
+5. Customization: KServe is highly customizable, allowing you to define how you want to serve your models, handle preprocessing and post-processing tasks, and even add custom logic to your serving containers.
+6. Integration with Kube Flow Pipelines: If you're using Kube Flow Pipelines for end-to-end machine learning workflows, KServe can be integrated seamlessly to deploy models as part of your pipeline.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:93]: I think Denis had a question as to whether Kube Flow was one word (KubeFlow) or not

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 20]
```diff
@@ -0,0 +1,153 @@
...
+
+The first step of model deployment is to define the service account with MinIO object store secrets.
+
+![](/img/fig-1a-define-service-acct.png "Figure 8. Defining service account with MinIO secret. ")
+
+Next, script the KServe yaml file to begin the inference service. 
+
+![](/img/fig-1b-inference-service.png "Figure 9. The inference service is defined. ")
+
+Next, a new model server must be created in Kube Flow and the yaml file uploaded. The inference service is created and once it is running, we can send our test data through REST APIs and get the response.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:105]: Change the last sentence to read:
"...once it is running, you can send your test data through REST APIs to..."

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 21]
```diff
@@ -0,0 +1,153 @@
...
+
+**Model Retraining**
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:107]: Use "Model retraining"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 22]
```diff
@@ -0,0 +1,153 @@
...
+
+Python programs need to be modified to run as a Kube Flow pipeline because each step runs as a containerized module. The whole cycle of data collection, preprocessing, and modeling are divided into separate components with each process isolated and running as independent containers inside individual Kubernetes pods. All the dependencies of each task must be mentioned inside the task due to the isolated nature of the environment. The artifacts that are used in the process, such as input data files and model objects, are saved in specific locations and referenced by the tasks. 
+
+To prepare the Kube Flow pipelines, we have to define each task as a function. First “read_data()” is defined to read the input file from the data volume on the Kube Flow pipeline.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:113]: Kube Flow question
To avoid the Royal We, try this:
"To prepare the KubeFlow pipelines, define each task as a function."

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 23]
```diff
@@ -0,0 +1,153 @@
...
+
+![](/img/fig-1-define-each-task.png "Figure 10. Each task is defined as a function that includes all necessary packages. ")
+
+Once defined, functions are converted into a Kube Flow component using “create_component_from_func()”.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:117]: Kube Flow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 24]
```diff
@@ -0,0 +1,153 @@
...
+
+![](/img/fig-2-converting-each-task.png "Figure 11. Each task is converted into a component that includes base line images and packages. ")
+
+Once each of the tasks are defined, the Kube Flow pipelines are defined to call each of the steps using DSL (Domain Specific Language).
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:121]: Kube Flow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 25]
```diff
@@ -0,0 +1,153 @@
...
+
+![](/img/fig3-define-the-pipeline.png "Figure 12. Pipelines are defined along with tasks in the appropriate sequence. ")
+
+When the Kube Flow pipeline is set up, the pipeline can run using the Kube Flow client where the required arguments and parameters are passed for the specific run.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:125]: Kube Flow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 26]
```diff
@@ -0,0 +1,153 @@
...
+
+![](/img/kubeflow-setup-run.png "Figure 13. Kube Flow client is created and executed with appropriate arguments. ")
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:127]: Kube Flow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md (hunk 8 above)
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:129]: Kube Flow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 27]
```diff
@@ -0,0 +1,153 @@
...
+
+In the Kube Flow UI, “stock_pipeline_run” would be triggered to provide a green-red status of each step as it executes. 
+
+![](/img/fig-5-simplified-pipeline.png "Figure 14. Status of each pipeline step and it's status. ")
+
+**Conclusion**
+
+From the above example, HPE Ezmeral Unified Analytics has significantly enhanced the end-to-end pipeline of financial service use cases by providing a comprehensive platform tailored to the specific needs of ML practitioners and data scientists. Here is a conclusion highlighting the benefits of HPE Ezmeral Unified Analytics:
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:135]: I'd recommend combining these two paragraphs to read something like this:

As you can see, HPE Ezmeral Unified Analytics has significantly enhanced the end-to-end pipeline of the financial service use cases by providing a comprehensive platform tailored to the specific needs of ML practitioners and data scientists. HPE Ezmeral Unified Analytics offers seamless integration of various data engineering and ML components, like Spark, EzPresto, Superset, MLflow, and Kube Flow. This integration streamlines the entire ML workflow, reducing friction between dissimilar stages and enables more efficient development of ML pipelines. 

In summary: (and then it goes to your bullets)

(or something like that at the end... feels like it's missing a few words at the very end. Also, you didn't need "Here is a conclusion highlighting the benefits of..."

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md (hunk 9 above)
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:137]: Use "Seamless integration:"

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 28]
```diff
@@ -0,0 +1,153 @@
...
+
+Data Exploration and Visualization: By seamlessly integrating Apache Superset into the Ezmeral ecosystem, organizations can harness the power of data-driven insights. Teams can easily access and visualize data, track experiment results, and make data-informed decisions to improve model quality and performance.
+
+Reproducibility: Kube Flow ensures reproducibility by allowing users to define and version their entire ML pipelines using HPE Ezmeral Unified Analytics. This means that every step, from data preparation to model deployment, can be tracked, versioned, and replicated easily, leading to better model accountability and auditability.
+
+Scalability: Kube Flow on HPE Ezmeral Unified Analytics excels at scaling applications. Kube Flow leverages this capability to scale ML workloads as needed, handling large datasets and complex models with ease. This scalability is crucial for training and serving models in real-world, production environments.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:143]: Kube Flow?

File: content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md [hunk 29]
```diff
@@ -0,0 +1,153 @@
...
+
+Model Serving: HPE Unified Analytics through Kube Flow Serving (KServe) simplifies model deployment and serving. It provides features like advanced deployment strategies, model versioning, and monitoring, making it easier to roll out new models, manage production deployments, and ensure model performance and reliability.
+
+Automated Workflow Orchestration: Kube Flow Pipelines simplifies the orchestration of ML workflows as code. With HPE Unified Analytics, this automation reduces manual effort, enhances reproducibility, and accelerates experimentation, resulting in more efficient model operations.
```
Comment (drensing) [inline content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md:147]: Use "Automated workflow orchestration:"
Kube Flow question
"Pipelines" should be "pipelines"
Question on HPE Unified Analytics or HPE Ezmeral Unified Analytics

**Review by drensing — COMMENTED** [review]
There you go! Second blog post reviewed.

All the best,
Dale
